container/root/etc/nginx/nginx.conf

```diff
@@ -88,7 +88,7 @@ http {
 
     log_format main '$remote_addr - $remote_user [$time_local] "$request" '
                     '$status $body_bytes_sent "$http_referer" '
-                    '"$http_user_agent" "$http_x_forwarded_for" NGINX_SERVER';
+                    '"$http_user_agent" "$http_x_client_ip" "$http_x_forwarded_for" NGINX_SERVER';
```

Comment:
wondering if this could be defined at the "server" level, instead of "HTTP", that way we could define the format in our service configuration